CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 ## Unreleased
 
 * Remove font_size option on contents list ([PR #1325](https://github.com/alphagov/govuk_publishing_components/pull/1325))
+* Revert 'Dont show breadcrumb item with no url' ([PR #1330](https://github.com/alphagov/govuk_publishing_components/pull/1330))
+* Add `collapse_on_mobile` flag which can be used to override default collapsing behaviour ([PR #1330](https://github.com/alphagov/govuk_publishing_components/pull/1330))
```

Comment:
This is outside of our conventions for changelog entries but I see why you've done it. Could we make the PR title and this line match?

Reply:
@andysellick This PR covers both changes, so I didn't want to make the PR title too specific otherwise it may imply that the revert isn't included here?